makefile

```diff
@@ -8,32 +8,33 @@ CHEATSHEETS   = pymapdl_cheat_sheet pyfluent_cheat_sheet pyaedt_API_cheat_sheet
 all: $(CHEATSHEETS)
 
 pymapdl_cheat_sheet:
-	latexmk -f -pdf -use-make cheat_sheets/mapdl_cheat_sheet/pymapdl_cheat_sheet.tex -cd -outdir=../../$(BUILD_DIR) -interaction=nonstopmode || true
+	latexmk -f -pdf -use-make -outdir=$(BUILD_DIR) cheat_sheets/mapdl_cheat_sheet/pymapdl_cheat_sheet.tex  -interaction=nonstopmode || true
 	convert -density 150 -scene 1 $(BUILD_DIR)/pymapdl_cheat_sheet.pdf $(BUILD_DIR)/pymapdl_cheat_sheet.png
 	(test -f $(BUILD_DIR)/pymapdl_cheat_sheet.pdf && echo pdf exists) || exit 1
 
 pyfluent_cheat_sheet:
-	latexmk -f -pdf -use-make cheat_sheets/pyfluent_cheat_sheet/pyfluent_cheat_sheet.tex -cd -outdir=../../$(BUILD_DIR) -interaction=nonstopmode || true
+	python3 -c "from scripts.generate_code_snippet import generate_code_examples; generate_code_examples('cheat_sheets/pyfluent_cheat_sheet/pyfluent_script.py', 'scripts/generated_scripts')"
```

Comment:
Nope, don't do it like this. Provide an entrypoint to the script. It should be like I proposed somewhere in my previous review :)

Reply:
Thanks @RobPasMue , adding that right now.